components/MdxComponents.tsx

```diff
@@ -126,6 +120,25 @@ const Blockquote: React.FunctionComponent<{ id: string }> = ({ children }) => {
   );
 };
 
+const H1 = ({ children }) => (
+  <h1 className="MuiTypography-root MuiTypography-h1" id={children.toLowerCase().replace(/ /g, "-")}>{children}</h1>
```

Comment:
we need to make sure that the children is a string. something like:

```typescript
typeof children === 'string' ? children.toLowerCase().replace(/ /g, "-") : ""
```

The latest build failed: https://github.com/dvargas92495/roamjs-com/actions/runs/4055445132/jobs/6978593292

Reply:
what about `toString()` https://github.com/dvargas92495/roamjs-com/pull/752